### PR TITLE
Admin/manage areas/show registered areas

### PR DESCRIPTION
**Pull request by tsato21**
Additions:
- Added showAreas function in AreasController
- Added UI in show.blade.php
- Added use for AreasController and route for showAreas function in web.php

### CODE REVIEW

**Review by KobeSergio — APPROVED** [review]

**Comment by KobeSergio** [comment]
Please resolve conflicts first so we can push to main @tsato21 :)

**Comment by tsato21** [comment]
@KobeSergio
I have resoved the confilicts and committed and pushed again.
Can you kindly check the udpates and merge this into the main?

File: app/Http/Controllers/admin/AreasController.php [hunk 1]
```diff
@@ -7,7 +7,44 @@
 
 class AreasController extends Controller
 {
+<<<<<<< HEAD
```
Comment (Ninit29) [inline app/Http/Controllers/admin/AreasController.php]: There was an issue when resolving the conflicts, please fix this

Comment (Ninit29) [reply app/Http/Controllers/admin/AreasController.php]: remove <<<<<<< HEAD 
remove =======
>>>>>>> main

Comment (YoshimiHELLo) [reply app/Http/Controllers/admin/AreasController.php]: @Ninit29 

I've fixed it. kindly confirm it please

**Pull request merged**